### ISSUE TITLE
question about the similar AP in KITTI test set and verification set

### ISSUE THREAD

**Issue by yjy4231** [SPengLiang/DID-M3D#12]
Hi, thanks for your great work and the great code.I'm having problems about your results. 
Use the pre-trained model you provide to test  the AP results(Car@3D IoU=0.7: 25.38  17.07  14.06) are in the KITTI validation set or the  KITTI test set?
I read the code  and think the results are in the KITTI validation set. But the AP results(Car@3D IoU=0.7: 22.98  16.12  14.03) in original paper, I find the results in my download paper are (24.40 16.29 13.75) in KITTI test set, the results are similar on KITTTI Benchmark. I also read other paper and found that the AP of the KITTI verification set is generally 3-5 points higher than that of the test set.
So, Your code has the similar AP in KITTI test set and verification set？
thanks！
![image](https://user-images.githubusercontent.com/52461074/197937291-40b6c23b-b371-4a9c-930e-b21e02834c85.png)

![image](https://user-images.githubusercontent.com/52461074/197936962-3e79a1fa-56ad-4b5c-80e9-98e5393ff212.png)

**Comment by SPengLiang** [comment]
Thanks for your interest in this work. 1. Yes, the pre-trained model is evaluated on the KITTI validation set. 2. According to our results, our work has a similar AP in KITTI test set and validation set. Actually, not all works have much higher results (3-5 points higher) on validation set compared to the ones on test set (e.g., MonoRCNN: 16.61/13.19/10.65 in val and 18.36/12.65/10.03 in test)).

**Comment by yjy4231** [comment]
Hi. bother you again!

I want to reproduce your provided result(Car@3D IoU=0.7[25.38 ,17.07, 14.06])，however, I have tried many times, and the best result is (Car@3D IoU=0.7[23.58 ,15.59, 13.39]), and the result fluctuates greatly. Is this normal?

I see the 'DLAUpv2' in your code, you provided result(Car@3D IoU=0.7[25.38 ,17.07, 14.06]) is training with 'DLAUp' or 'DLAUpv2'? or there are other settings different from your open source code？

Thanks!

**Comment by SPengLiang** [comment]
It seems that your results are lower than expected, which is weird. We use the default setting in the released code (i.e., 'DLAUp').
Using the released code, some other people can obtain similar results to the released ckpt (17.07 moderate AP_3D), e.g. 17.44 moderate AP_3D in https://github.com/SPengLiang/DID-M3D/issues/9 and 16.83 AP_3D in https://github.com/SPengLiang/DID-M3D/issues/5).

Can you check your torch/CUDA version? A much lower version may cause performance degradation (https://github.com/SPengLiang/DID-M3D/issues/7).
